client/src/components/users/UserCard.vue

```diff
@@ -4,12 +4,15 @@
   <ion-item
     class="user-card-item ion-no-padding"
     :detail="false"
-    :class="{ selected: user.isSelected, 'no-padding-end': !user.isSelected }"
+    :class="{ selected: user.isSelected, 'no-padding-end': !user.isSelected, revoked: user.isRevoked() }"
     @click="$emit('click', $event, user)"
     @mouseenter="isHovered = true"
     @mouseleave="isHovered = false"
   >
-    <div class="card-checkbox">
+    <div
+      class="user-card-checkbox"
+      v-if="!user.isRevoked()"
```

Comment:
In list mode you can select a revoked user. If you select one and switch to grid mode, the user will still be selected.
I don't think it makes much sense to be able to select a revoked user so I'm fine with this change, but it should also be the case in list mode.

Reply:
Yes, we probably don't need to select the revoked user, each revoked should be edit independently